source/classes.tex

```diff
@@ -6628,8 +6628,8 @@
 between $\tcode{x}_i$ and $\tcode{y}_i$
 yields a result value $\tcode{v}_i$ where $\tcode{v}_i \mathrel{\tcode{!=}} 0$,
 contextually converted to \tcode{bool}, yields \tcode{true};
-\tcode{V} is a copy of $\tcode{v}_i$.
-If no such index exists, \tcode{V} is
+the return value is a copy of $\tcode{v}_i$.
```

Comment:
Perhaps here too we can replace the semicolon with a period (the sentence is already long enough), and then rephrase the conclusion to say "return value" only once?

Reply:
@brevzin ?

Reply:
@brevzin Ping :-)

Reply:
I'm not sure what you mean. Rephrase how?

Reply:
Oh, there's more words after this comment lol. So something like:

>The return value is a copy of $\tcode{v}_i$ is such an index exists, otherwise \tcode{static_cast<R>(std::strong_ordering::equal)}.

?

Reply:
Yes, or "The return value is a copy of v_i if such an index exists, and static_cast... otherwise." See comment above.